examples/xegpu_matmul/matmul.py

```diff
@@ -375,9 +382,14 @@ def parse_cli():
             has_relu=args.relu,
         )
 
-        if args.dump_kernel or args.dump_schedule:
+        if args.dump_schedule:
+            schedule_module = wload.schedule_module(
+                stop_at_stage=args.dump_payload, parameters=params
+            )
+            print(schedule_module)
+        elif args.dump_payload:
```

Comment:
It would be good if we could still dump both the payload and schedule simultaneously if so desired.